tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -863,6 +868,20 @@ class TransformBasedCpuFusionStrategy : public FusionStrategy {
   }
 };
 
+class SparseOpCpuFusionStrategy : public FusionStrategy {
```

Comment:
nit: better to add a filecheck based ut?

Reply:
done